src/Patronage.DataAccess/Services/ProjectService.cs

```diff
@@ -3,101 +3,152 @@
                 .ToList();
 
-            var projectsDto = _mapper.Map<List<ProjectDto>>(projects);
+
+            var projectsDto = new List<ProjectDto>();
+
+            foreach (var project in projects)
+            {
+                projectsDto.Add(new ProjectDto
+                {
+                    Id = project.Id,
+                    Name = project.Name,
+                    Alias = project.Alias,
+                    Description = project.Description,
+                    IsActive = project.IsActive,
+                    CreatedOn = project.CreatedOn,
+                    ModifiedOn = project.ModifiedOn
+                });
+            }
 
             return projectsDto;
         }
 
 
 
-        public ProjectDto GetById(int id)
+        public async Task<ProjectDto> GetById(int id)
         {
             var project = _dbContext
                 .Projects
                 .FirstOrDefault(p => p.Id == id);
 
             if (project is null) return null;
 
-            var projectDto = _mapper.Map<ProjectDto>(project);
+
+            ProjectDto projectDto = new ProjectDto
+            {
+                Id = project.Id,
+                Name = project.Name,
+                Alias = project.Alias,
+                Description = project.Description,
+                IsActive = project.IsActive,
+                CreatedOn = project.CreatedOn,
+                ModifiedOn = project.ModifiedOn
+            };
 
             return projectDto;
         }
 
 
 
-        public bool Update(int id, CreateOrUpdateProjectDto projectDto)
+        public async Task<bool> Update(int id, UpdateProjectDto projectDto)
         {
             var project = _dbContext
                 .Projects
                 .FirstOrDefault(p => p.Id == id);
 
             if (project is null) return false;
 
-            _mapper.Map(projectDto, project);
+            project.Name = projectDto.Name;
+            project.Alias = projectDto.Alias;
+            project.Description = projectDto.Description;
+            project.IsActive = projectDto.IsActive;
 
-            _dbContext.SaveChanges();
+            await _dbContext.SaveChangesAsync();
 
             return true;
         }
 
 
 
-        public bool LightUpdate(int id, PartialProjectDto projectDto)
+        public async Task<bool> LightUpdate(int id, PartialProjectDto projectDto)
         {
             var project = _dbContext
                 .Projects
                 .FirstOrDefault(p => p.Id == id);
 
             if (project is null) return false;
 
-            _mapper.Map(projectDto, project);
 
-            _dbContext.SaveChanges();
+            if (projectDto.Description == null) project.Description = project.Description;
+            else if (projectDto.Description?.Data == null && projectDto.Description != null) project.Description = null;
+            else project.Description = projectDto.Description.Data;
```

Comment:
```suggestion
            project.Description = projectDto.Description ?? project.Description.Data;
```
unsure how this will behave if dto is initialized with value 'null' but that's much clearer syntax.